src/transformers/convert_graph_to_onnx.py

```diff
@@ -37,9 +38,41 @@ def __init__(self):
         self.add_argument("--opset", type=int, default=11, help="ONNX opset to use")
         self.add_argument("--check-loading", action="store_true", help="Check ONNX is able to load the model")
         self.add_argument("--use-external-format", action="store_true", help="Allow exporting model >= than 2Gb")
+        self.add_argument("--quantize", action="store_true", help="Quantize the neural network to be run with int8")
         self.add_argument("output")
 
 
+def generate_identified_filename(filename: Path, identifier: str) -> Path:
+    """
+    Append an string-identifier at the end (before the extension,  if any) to the provided filepath.
+    Can be used to
+    :param filename:
+    :param identifier:
+    :return:
+    """
```

Comment:
Maybe at one point this doc could be updated to fit the google style that we use everywhere else!

Reply:
Yes ofc, will update the docstrings, sorry for this ...